generic/tests/netperf.py

```diff
@@ -220,6 +217,21 @@ def _pin_vm_threads(vm, node):
         else:
             logging.info("FIXME: support to enable MQ for Windows guest!")
 
+    if params.get("server_private_ip"):
+        cmd = "systemctl stop NetworkManager.service"
+        cmd += "ifconfig %s %s up" % (ethname, params.get("server_private_ip"))
```

Comment:
```suggestion
        cmd += " && ifconfig %s %s up" % (ethname, params.get("server_private_ip"))
```
And we better turn from `ifconfig` to `ip link/addr` in the future.

Reply:
@PaulYuuu I have updated. please help review. thanks